rest-api/controllers/pokemon_feed.go

```diff
@@ -102,6 +106,81 @@ func Api_feed(w http.ResponseWriter, r *http.Request) {
 		json.NewEncoder(w).Encode(fs)
 	}
 }
+func worker_add(id int, jobs <-chan Pokemon, results chan<- Pokemon) {
+	for j := range jobs {
+		if total > 0 && total_per_worker[id-1] > 0 {
+			fmt.Println("worker", id, "started  job", j)
+			fmt.Println("worker", id, "finished job", j)
+			results <- j
+		}
+		total--
+		total_per_worker[id-1] = total_per_worker[id-1] - 1
+	}
+}
+func Worker(w http.ResponseWriter, r *http.Request) {
+	jobs := make(chan Pokemon, 100)
+	results := make(chan Pokemon, 100)
+
+	items_per_workers := r.URL.Query().Get("items_per_workers")
+	intValue_per_worker, _ := strconv.Atoi(items_per_workers)
+
+	type_value := r.URL.Query().Get("type") //odd or even
+
+	items := r.URL.Query().Get("items")
+
+	items_value, _ := strconv.Atoi(items)
+
+	total = items_value
+	fmt.Printf(items_per_workers)
```

Comment:
Printf receives a format string as first argument, maybe you were trying to use Print or Println? 
https://pkg.go.dev/fmt#Printf

Reply:
thanks , I'm using all the feedback on this kind of things to change all the file and follow the best practices on further developments.